index.html

```diff
@@ -320,18 +320,17 @@
           view in degrees.
         </p>
         <p>
-          A <a>depth map</a> has an associated <dfn>unit</dfn> which is a
-          string. It represents the <a>active depth map unit</a>.
+          A <a>depth map</a> has an associated <dfn>precision</dfn> which is a
+          string. It represents the <a>depth precision</a> (one of the
+          <a>possible depth precisions</a>).
         </p>
         <p>
           A <a>depth map</a> has an associated <dfn>near value</dfn> which is a
-          double. It represents the minimum range in <a>active depth map
-          unit</a>s.
+          double. It represents the minimum range in millimeters.
```

Comment:
Wondering, would it make more sense to go with the base unit (meters) since milli is literally .001 and this number is double precision, or is millimeter a defacto enough standard here already that it would be weird?